fn_phish_tank/fn_phish_tank/components/fn_phish_tank_submitt_url.py

```diff
@@ -0,0 +1,93 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+# pragma pylint: disable=unused-argument, no-self-use
+"""Function implementation"""
+
+import logging
+from resilient_circuits import ResilientComponent, function, handler, StatusMessage, FunctionResult, FunctionError
+import fn_phish_tank.util.selftest as selftest
+from resilient_lib import ResultPayload
+from fn_phish_tank.util.phish_tank_helper import *
+
+
+class FunctionComponent(ResilientComponent):
+    """Component that implements Resilient function 'fn_phish_tank_submitt_url"""
+
+    def __init__(self, opts):
+        """constructor provides access to the configuration options"""
+        super(FunctionComponent, self).__init__(opts)
+        self.options = opts.get("fn_phish_tank", {})
+        selftest.selftest_function(opts)
```

Comment:
self can be removed

Reply:
which self i needs to remove from here, since only configuration file parameters are assigned to self.options.

Reply:
sorry. selftest.selftest_function(opts)

Reply:
ok